src/System.DirectoryServices/src/project.json

```diff
@@ -3,17 +3,24 @@
     "netstandard1.7": {
       "dependencies": {
         "Microsoft.NETCore.Platforms": "1.1.0",
-        "System.Collections": "4.4.0-beta-24910-02",
-        "System.Collections.NonGeneric": "4.4.0-beta-24910-02",
-        "System.IO": "4.4.0-beta-24910-02",
-        "System.IO.FileSystem": "4.4.0-beta-24910-02",
-        "System.IO.FileSystem.AccessControl": "4.4.0-beta-24910-02",
-        "System.Runtime": "4.4.0-beta-24910-02",
-        "System.Runtime.Extensions": "4.4.0-beta-24910-02",
-        "System.Security.AccessControl": "4.4.0-beta-24910-02",
-        "System.Security.Permissions": "4.4.0-beta-24910-02",
-        "System.Security.Principal": "4.4.0-beta-24910-02",
-        "System.Security.Principal.Windows": "4.4.0-beta-24910-02"
+        "System.Collections": "4.4.0-beta-24814-01",
```

Comment:
The CI/build system is pretty particular with its package versions. Every time the repo "master version" is updated, existing PR's that add new packages will need to explicitly update the package versions to match so the build doesn't complain. 

This is the cause of the errors you're currently seeing on Github and the reason why you wouldn't see them in a local build unless you rebased against the latest master.